### ISSUE TITLE
Unable to create virtual directory on application

### ISSUE THREAD

**Issue by fdzedzy** [chef-cookbooks/iis#369]
### Cookbook version
6.5.2

### Chef-client version
13.0.118

### Platform Details
Windows 2012R2 AWS EC2

### Scenario:
Add a virtual directory to an application.  This was working prior to [PR 363](https://github.com/chef-cookbooks/iis/pull/363)

### Steps to Reproduce:
```
iis_vdir 'Creating vDir /vdir_test2 in app' do
  application_name 'Default Web Site/app_test'
  path '/vdir_test2'
  physical_path "#{node['iis']['docroot']}/app_test/vdir_test2"
  action [:add, :config]
end
```

### Expected Result:
Virtual directory created inside the app.

### Actual Result:
```
FATAL: Mixlib::ShellOut::ShellCommandFailed: iis_vdir[Creating vDir /vdir_test2 in app] (test::vdir line 74) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0, 42, 183], but received '50'
       ---- Begin output of C:\Windows\System32\inetsrv\appcmd.exe add vdir /app.name:"Default Web Site/app_test/" /path:"/vdir_test2" /physicalPath:"C:\inetpub\wwwroot\app_test\vdir_test2" /logonMethod:ClearText /allowSubDirConfig:true /commit:\"MACHINE/WEBROOT/APPHOST\" ----
       STDOUT: ERROR ( message:Must use exact identifer for VDIR object with verb ADD. )
```

I have a unit test that will recreate the issue.  I will try to look at the referenced PR where this might have broken to see if I can figure out what the fix it.

**Comment by EasyAsABC123** [comment]
on it

**Comment by EasyAsABC123** [comment]
Resolved by #370

**Comment by fdzedzy** [comment]
@EasyAsABC123  Thanks for the quick response!

**Comment by EasyAsABC123** [comment]
@fdzedzy never a problem, thanks for finding this.  As the test suite grows so does our coverage

**Comment by EasyAsABC123** [comment]
Resolved in #371 release v6.5.3
